draft-ietf-tls-tls13.md

```diff
@@ -3468,30 +3468,26 @@ One of the content types supported by the TLS record layer is the
 alert type.  Like other messages, alert messages are encrypted as
 specified by the current connection state.
 
-Alert messages convey the severity of the message (warning or fatal)
-and a description of the alert. Warning-level messages are used to
-indicate orderly closure of the connection (see {{closure-alerts}}).
-Upon receiving a warning-level alert, the TLS implementation SHOULD
-indicate end-of-data to the application and, if appropriate for
-the alert type, send a closure alert in response.
-
-Fatal-level messages are used to indicate abortive closure of the
-connection (See {{error-alerts}}). Upon receiving a fatal-level alert,
+Alert messages convey a description of the alert. Alerts listed in
+{{closure-alerts}} are used to indicate an orderly close of the current
+connection state and must be handled as specified.
+
+Alerts listed in {{error-alerts}} are used to indicate abortive
+closure of the connection. Upon receiving an error alert,
 the TLS implementation SHOULD indicate an error to the application and
 MUST NOT allow any further data to be sent or received on the
 connection.  Servers and clients MUST forget keys and secrets
 associated with a failed connection. Stateful implementations of
 session tickets (as in many clients) SHOULD discard tickets associated
-with failed connections.
+with failed connections. All unknown alert types MUST be treated as
+error alerts.
 
-All the alerts listed in {{error-alerts}} MUST be sent as fatal and
-MUST be treated as fatal regardless of the AlertLevel in the
-message. Unknown alert types MUST be treated as fatal.
+Prior versions of TLS used the "legacy_alert_level" field to indicate the
+severity of the alert. This field should be set to {2} for all alerts,
```

Comment:
I prefer "This field **MUST** be set to 2 for all alerts", because we use "MUST" for most other deprecated fields, such as ClientHello.legacy_version, ClientHello.legacy_compression_methods,  TLSPlaintext.legacy_record_version.